File: AES/aes.js
const crypto = require("crypto");

const algorithm = "aes-128-cbc";
const _iv_ = "BBBBBBBBBBBBBBBB";

function aes_encrypt(data, key) {
    const cipher = crypto.createCipheriv(algorithm, key, _iv_);
    let enc = cipher.update(data, "utf8", "base64") + cipher.final('base64');
    return enc;
}

function aes_decrypt(enc, key) {
    const decipher = crypto.createDecipheriv(algorithm, key, _iv_);
    var data = decipher.update(enc, "base64", "utf8");
    return data + decipher.final("utf8");
}

module.exports = {
    aes_encrypt,
    aes_decrypt
}

File: AES/aes.py
import base64 
from Crypto.Cipher import AES
from Crypto.Util.Padding import pad,unpad

#CBC with Fix IV

#FIX IV
_iv_ =  'BBBBBBBBBBBBBBBB'.encode('utf-8') #16 char for AES128

def aes_encrypt(data,key):
        data= pad(data.encode(),16)
        cipher = AES.new(key.encode('utf-8'),AES.MODE_CBC,_iv_)
        return base64.b64encode(cipher.encrypt(data))

def aes_decrypt(enc,key):
        enc = base64.b64decode(enc)
        cipher = AES.new(key.encode('utf-8'), AES.MODE_CBC, _iv_)
        return unpad(cipher.decrypt(enc),16)


if __name__ == '__main__':
    data = 'I love you'
    key = 'AAAAAAAAAAAAAAAA' #16 char for AES128

    encrypted = aes_encrypt(data,key)
    print('encrypted CBC base64 : ',encrypted.decode("utf-8", "ignore"))

    decrypted = aes_decrypt(encrypted,key)
    print('data: ', decrypted.decode("utf-8", "ignore"))